specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-09-01/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1199 @@
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "A container holding only the Tags for a resource, allowing the user to update the tags on a PrivateLinkScope instance."
+    },
+    "AzureMonitorPrivateLinkScope": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties that define a Azure Monitor PrivateLinkScope resource.",
+          "$ref": "#/definitions/AzureMonitorPrivateLinkScopeProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System data",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "description": "An Azure Monitor PrivateLinkScope definition."
+    },
+    "AzureMonitorPrivateLinkScopeProperties": {
+      "description": "Properties that define a Azure Monitor PrivateLinkScope resource.",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Succeeded",
+                "description": "The state is Succeeded.",
+                "name": "Succeeded"
+              },
+              {
+                "value": "Failed",
+                "description": "The state is Failed.",
+                "name": "Failed"
+              },
+              {
+                "value": "Canceled",
+                "description": "The state is Canceled.",
+                "name": "Canceled"
+              },
+              {
+                "value": "Provisioning",
+                "description": "The state is Provisioning.",
+                "name": "Provisioning"
+              }
+            ]
+          },
+          "description": "Current state of this PrivateLinkScope: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Provisioning ,Succeeded, Canceled and Failed.",
+          "readOnly": true
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "description": "List of private endpoint connections.",
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v2/privatelinks.json#/definitions/PrivateEndpointConnection"
```

Comment:
>PrivateEndpointConnection [](http://example.com/codeflow?start=103&length=25)

This is defined as a separate proxy resource. Is the NRP contract forcing this to be present inside the properties bag of the privateLinkScope resource type?